keyboards/lily58/keymaps/druotoni/keymap_french.h

```diff
@@ -0,0 +1,165 @@
+/* Copyright 2015-2016 Jack Humbert
```

Comment:
This file should be deleted, as its just <https://github.com/qmk/qmk_firmware/blob/master/quantum/keymap_extras/keymap_french.h>